ui/src/criteria/concept.tsx

```diff
@@ -66,25 +69,21 @@ export class ConceptCriteria extends Criteria {
     };
   }
 
-  selected = new Array<GridRowData>();
+  selected = new Array<Selection>();
 }
 
-const fetchedColumns: GridColDef[] = [
-  { field: "concept_name", headerName: "Concept Name", width: 400 },
-  { field: "concept_id", headerName: "Concept ID", width: 100 },
-  { field: "domain_id", headerName: "Domain", width: 100 },
-  {
-    field: "standard_concept",
-    headerName: "Source/Standard",
-    width: 140,
-    valueFormatter: (params: GridValueFormatterParams) =>
-      !params.value ? "Source" : "Standard",
-  },
-  { field: "vocabulary_id", headerName: "Vocab", width: 120 },
-  { field: "concept_code", headerName: "Code", width: 120 },
-  // TODO(tjennison): Add support for counts to the API.
-  //{ field: "roll_up_count", headerName: "Roll-up Count", width: 100 },
-  //{ field: "item_count", headerName: "Item Count", width: 100 },
+const fetchedColumns: TreeGridColumn[] = [
+  { key: "concept_name", width: "100%", title: "Concept Name" },
+  { key: "concept_id", width: 120, title: "Concept ID" },
+  { key: "domain_id", width: 100, title: "Domain" },
+  { key: "standard_concept", width: 180, title: "Source/Standard" },
+  { key: "vocabulary_id", width: 120, title: "Vocab" },
+  { key: "concept_code", width: 120, title: "Code" },
+];
+
+const allColumns: TreeGridColumn[] = [
+  ...fetchedColumns,
+  { key: "view_hierarchy", width: 160, title: "View Hierarchy" },
```

Comment:
SHould this key be special, e.g.  an exported constant from TreeGrid? If it's not a special value, how is the TreeGrid recognizing where to put the heirarchy expansion handler?

Reply:
The TreeGrid doesn't know anything about it. TreeGridData accepts a JSX.Element instead of string. It's filled in on line 134, though without a handler attached.

Something along those lines came up in one of your comments on the previous commit and I didn't think to mention this. There's overlap with prefixElements but this replaces the whole cell which would require handling the layout and formatting of the text. I think this is a good balance for what we need now but we'll see how it evolves.